docs/reference/release-notes/7.11.asciidoc

```diff
@@ -153,6 +153,15 @@ Also see <<breaking-changes-7.11,Breaking changes in 7.11>>.
   downgrades are not safe. Elasticsearch 7.11.1 will contain a fix for this
   issue. For more details, see {es-issue}68838[#68838].
 
+* If autoscaling is enabled for machine learning, the administrator of the cluster
+  should increase the cluster setting `xpack.ml.max_open_jobs` to the maximum value of `512`. This allows autoscaling
+  to run reliably as it relies only on assigning jobs via memory. Having
```

Comment:
I *think* the "only" is in the wrong place here. Is the key point that it's "only assigning jobs" (instead of other tasks) or "only via memory" (as opposed to something less performant?)
```suggestion
  to run reliably as it relies on assigning jobs only via memory. Having
```